src/style.css

```diff
@@ -1,3 +1,80 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+ul,
+li,
+nav,
+header,
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
you added box-sizing for ul,li,nav,header,body it is redundant, add once in `*` for all elements